Entities/Items/Sponge/SpongeDryInAir.as

```diff
@@ -11,7 +11,10 @@ void onInit(CBlob@ this)
 
 void onTick(CBlob@ this)
 {
-    if (not this.isInWater())
+	Vec2f bottom_pos = this.getPosition() + Vec2f(0, this.getHeight() / 2);
+	CMap@ map = getMap();
+
+	if (!map.isInWater(bottom_pos))
```

Comment:
Fix the indent.

While this might fix them drying in water, if water comes from the top or from the sides, this check will still report as it being in air.

I would instead do something like:
- Have we not removed water in the last 15(?) ticks
- Our velocity is 0 (e.g. we're on land) & not attached to anything

Reply:
Ok, I will do something.